package.xml

```diff
@@ -16,14 +16,14 @@
 
   <license>BSD</license>
 
+  <url type="website">http://ros.org/wiki/laser_geometry</url>
```

Comment:
Sorry, I missed this earlier.  We shouldn't point to the wiki anymore, but instead:

```suggestion
  <url type="website">http://github.com/ros-perception/laser_geometry</url>
```